drivers/entropy/Kconfig.nrf5

```diff
@@ -29,28 +29,30 @@ config ENTROPY_NRF5_THR_THRESHOLD
 	range 4 255
 	help
 	  Low-water threshold in bytes to trigger entropy generation for
-	  thread mode consumers.
+	  thread mode consumers. As soon as the currently available bytes go
+	  below this mark hardware entropy generation will be started.
```

Comment:
Nitpicking: "As soon as there are less available bytes than this threshold, hardware entropy generation will start"

Reply:
ok will rephrase a bit